### PR TITLE
[Master] [Feature #51815] init project

### PR DESCRIPTION
**Pull request by thonght-3051**

### CODE REVIEW

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# MySQL. Versions 5.5.8 and up are supported.
```
Comment (ghost) [inline config/database.yml]: ignore file này đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by thonght-3051** [comment]
ready

**Review by ghost — COMMENTED** [review]
thêm file database.yml.example

**Review by dangthibichngocb — APPROVED** [review]

**Comment by thonght-3051** [comment]
ready

**Pull request merged**